src/components/Select/SelectT.tsx

```diff
@@ -7,59 +7,67 @@ import Icon from '../Icon/Icon';
 import IconTooltip from '../IconTooltip';
 import { isWhitespaces } from '../../utils';
 
+// Styles for overriding select dropdown
+export interface SelectStyles {
+  inputContainer?: Record<string, any>;
+  optionsContainer?: Record<string, any>;
+}
+
 export interface SelectTProps<T> {
-  onChange: (newValue: T) => void;
-  label?: string;
-  disabled?: boolean;
-  id?: string;
   className?: string;
-  helperText?: string | string[];
-  placeholder?: string;
+  disabled?: boolean;
+  displayLabel: (option: any) => string;
+  editable?: boolean;
   errorText?: string | string[];
-  warningText?: string | string[];
-  selectedValue?: T;
-  readonly?: boolean;
-  options?: T[];
+  fixedMenu?: boolean;
   fullWidth?: boolean;
+  helperText?: string | string[];
   hideArrow?: boolean;
+  id?: string;
+  isEqual: (A: T, B: T) => boolean;
+  label?: string;
   onBlur?: () => void;
+  onChange: (newValue: T) => void;
   onFocus?: () => void;
-  fixedMenu?: boolean;
-  isEqual: (A: T, B: T) => boolean;
+  options?: T[];
+  placeholder?: string;
+  readonly?: boolean;
   renderOption: (option: T) => React.ReactNode;
+  required?: boolean;
+  selectedValue?: T;
+  selectStyles?: SelectStyles;
   toT: (input: string) => T;
-  displayLabel: (option: any) => string;
   tooltipTitle?: TooltipProps['title'];
-  editable?: boolean;
-  required?: boolean;
+  warningText?: string | string[];
```

Comment:
Besides the addition of `selectStyles?: SelectStyles;`, this is just an alphabetical reordering